src/bp-templates/bp-nouveau/buddypress/members/single/settings/profile.php

```diff
@@ -23,6 +23,11 @@
 		<?php
 		while ( bp_profile_groups() ) :
 			bp_the_profile_group();
+			$group_id	= bp_get_the_profile_group_id();
+			// Check if Current Group is repeater if YES then get number of fields inside current group.
+			$is_group_repeater_str = bp_xprofile_get_meta( $group_id, 'group', 'is_repeater_enabled', true );
+			$is_group_repeater     = ( 'on' === $is_group_repeater_str ) ? true : false;
+			$group_url	= esc_url( trailingslashit( bp_displayed_user_domain() . bp_get_profile_slug() . '/edit/group/' . $group_id ) );
```

Comment:
@krunal-buddyboss Make a tabbing proper follow phpcs.

Reply:
@chetansatasiya  applied the changes that suggested. Thanks